### PR TITLE
Update docs-theme version

### PR DESCRIPTION
**Pull request by lucasbaquinoo**
@shuding I updated the documentation with the latest versions of the two libraries in alpha. I've hidden the Bleed page because of the styles protruding from the side menu. Perhaps something to be seen later specifically in the library.

### CODE REVIEW

**Comment by lucasbaquinoo** [comment]
@shuding this PR is related with the issue #477

**Comment by B2o5T** [comment]
This PR contains a bunch of unnecessary things

**Comment by lucasbaquinoo** [comment]
> This PR contains a bunch of unnecessary things

@B2o5T  this is my contribution to the open-source universe. Could you help me identify and adjust the unnecessary parts?

File: .prettierignore [hunk 1]
```diff
@@ -1,2 +0,0 @@
-.next
-node_modules
```
Comment (B2o5T) [inline .prettierignore:2]: why did you remove this file?

File: custom.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+code {
+  @apply text-sm;
+}
```
Comment (B2o5T) [inline custom.css:3]: why do you overwrite `nextra-theme-docs` styles?

File: custom.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+.dark .invert-on-dark {
+  filter: invert(1) brightness(1.8);
+}
```
Comment (B2o5T) [inline custom.css:7]: what is the purpose of this style?

File: next.config.js [hunk 4]
```diff
@@ -1,6 +1,13 @@
...
+  theme: "nextra-theme-docs",
+  themeConfig: "./theme.config.js",
+  unstable_flexsearch: true,
   unstable_staticImage: true,
-})
-module.exports = withNextra()
+});
+
+module.exports = withNextra({
+  // reactStrictMode: true,
```
Comment (B2o5T) [inline next.config.js:9]: it's better to enable strict mode
```suggestion
  reactStrictMode: true,
```

File: package.json [hunk 5]
```diff
@@ -6,22 +6,34 @@
   "scripts": {
     "dev": "next",
     "start": "next start",
-    "build": "next build"
+    "build": "next build ",
```
Comment (B2o5T) [inline package.json:9]:
```suggestion
    "build": "next build",
```

File: package.json [hunk 6]
```diff
@@ -6,22 +6,34 @@
...
+    "lint": "next lint",
+    "schema": "ts-json-schema-generator -f tsconfig.json -p ./schema.d.ts -o ./public/schema.json -t Schema --minify"
```
Comment (B2o5T) [inline package.json:11]: what the purpose of this script?

File: package.json [hunk 7]
```diff
@@ -6,22 +6,34 @@
...
   },
   "author": "Shu Ding",
   "license": "MIT",
   "dependencies": {
-    "next": "^12.1.0",
-    "nextra": "^1.1.0",
-    "nextra-theme-docs": "^1.2.2",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1"
+    "next": "12.1.6",
```
Comment (B2o5T) [inline package.json:16]: you can use latest Next.js
```suggestion
    "next": "12.2.0",
```

File: package.json [hunk 8]
```diff
@@ -6,22 +6,34 @@
...
+    "nextra": "2.0.0-alpha.56",
+    "nextra-theme-docs": "2.0.0-alpha.59",
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
```
Comment (B2o5T) [inline package.json:20]: same you can use React 18

File: package.json [hunk 9]
```diff
@@ -6,22 +6,34 @@
...
   },
   "devDependencies": {
-    "prettier": "^2.0.5"
+    "@babel/core": "7.16.7",
+    "@types/node": "^18.0.1",
+    "@types/react": "17.0.37",
+    "autoprefixer": "10.2.6",
+    "csstype": "3.1.0",
```
Comment (B2o5T) [inline package.json:27]: why do we need it?

File: package.json [hunk 10]
```diff
@@ -6,22 +6,34 @@
...
+    "eslint": "8.10.0",
+    "eslint-config-next": "12.1.6",
+    "eslint-config-prettier": "8.4.0",
```
Comment (B2o5T) [inline package.json:30]: do you use it really? I don't think so

File: package.json [hunk 11]
```diff
@@ -6,22 +6,34 @@
...
+    "postcss": "8.3.5",
+    "tailwindcss": "3.1.3",
+    "typescript": "4.7.4"
   },
   "prettier": {
-    "semi": false,
-    "singleQuote": true
+    "embeddedLanguageFormatting": "off",
+    "htmlWhitespaceSensitivity": "strict"
```
Comment (B2o5T) [inline package.json:37]: Why did you remove previous configuration options and added your own?

File: pages/_app.js [hunk 12]
```diff
@@ -1,5 +1,13 @@
...
+import "../custom.css";
 
 export default function Nextra({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  const getLayout = Component.getLayout || ((page) => page);
+
+  return getLayout(
+    <>
+      <Component {...pageProps} />
+    </>
```
Comment (B2o5T) [inline pages/_app.js:11]:
```suggestion
      <Component {...pageProps} />
```
unneeded fragment

File: pages/_document.js [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+        <body>
+          <Main />
+          <NextScript />
+        </body>
+      </Html>
+    );
+  }
+}
+
+export default MyDocument;
```
Comment (B2o5T) [inline pages/_document.js:32]: Why did you add this file? Did you see that any example of Nextra doesn't use `_document` file, why did you add it?

File: pages/_document.js [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+      <Html>
+        <Head>
+          <link rel="preconnect" href="https://fonts.googleapis.com" />
+          <link
+            rel="preconnect"
+            href="https://fonts.gstatic.com"
+            crossOrigin="true"
+          />
+          <link
+            href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700;800&display=optional"
```
Comment (B2o5T) [inline pages/_document.js:19]: why did you decide to add a custom font?

File: tailwind.config.js [hunk 15]
```diff
@@ -0,0 +1,49 @@
+const colors = require("tailwindcss/colors");
+
+module.exports = {
+  content: [
+    "./components/**/*.js",
+    "./components/**/*.tsx",
```
Comment (B2o5T) [inline tailwind.config.js:6]:
```suggestion
    "./components/**/*.{js,tsx}",
```

File: tailwind.config.js [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+    "./nextra-theme-docs/**/*.js",
+    "./nextra-theme-docs/**/*.tsx",
+    "./nextra-theme-docs/**/*.css",
```
Comment (B2o5T) [inline tailwind.config.js:9]:
```suggestion
    "./nextra-theme-docs/**/*.{js,tsx}",
```

File: tailwind.config.js [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+    "./pages/**/*.md",
+    "./pages/**/*.mdx",
+    "./pages/**/*.tsx",
```
Comment (B2o5T) [inline tailwind.config.js:12]:
```suggestion
    "./pages/**/*.{md,mdx,tsx}",
```

File: tailwind.config.js [hunk 18]
```diff
@@ -0,0 +1,49 @@
...
+    "./theme.config.js",
+    "./styles.css",
```
Comment (B2o5T) [inline tailwind.config.js:14]: NEVER include `css` files in content field https://github.com/shuding/nextra/pull/495

File: tailwind.config.js (hunk 16 above)
Comment (B2o5T) [inline tailwind.config.js:9]: NEVER include `css` files in content field https://github.com/shuding/nextra/pull/495

File: tailwind.config.js [hunk 19]
```diff
@@ -0,0 +1,49 @@
...
+        mono: [
+          "Menlo",
+          "Monaco",
+          "Lucida Console",
+          "Liberation Mono",
+          "DejaVu Sans Mono",
+          "Bitstream Vera Sans Mono",
+          "Courier New",
+          "monospace",
+        ],
```
Comment (B2o5T) [inline tailwind.config.js:29]: Is it really necessary?

File: tailwind.config.js [hunk 20]
```diff
@@ -0,0 +1,49 @@
...
+        orange: colors.orange,
+        green: colors.green,
+        red: colors.red,
+        yellow: colors.yellow,
+      },
+      screens: {
+        sm: "640px",
+        md: "768px",
+        lg: "1024px",
+        betterhover: { raw: "(hover: hover)" },
```
Comment (B2o5T) [inline tailwind.config.js:44]: does it is used somewhere?

File: theme.config.js [hunk 21]
```diff
@@ -1,67 +1,98 @@
...
+  logo: function LogoActual() {
+    return (
+      <>
+        <span className="font-extrabold">Nextra</span>
+        <span className="mr-2 ml-2 text-gray-500 font-normal hidden md:inline">
+          The Next Site Builder
+        </span>
+      </>
+    );
+  },
```
Comment (B2o5T) [inline theme.config.js:27]: Why did you refactor it to react component? what is the purpose?
```suggestion
  logo: (
      <>
        <span className="font-extrabold">Nextra</span>
        <span className="mr-2 ml-2 text-gray-500 font-normal hidden md:inline">
          The Next Site Builder
        </span>
      </>
    );
```

File: theme.config.js [hunk 22]
```diff
@@ -1,67 +1,98 @@
...
+  head: function Head({ title, meta }) {
```
Comment (B2o5T) [inline theme.config.js:28]:
```suggestion
  head({ title, meta }) {
```

File: package.json [hunk 23]
```diff
@@ -6,22 +6,34 @@
...
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1"
+    "next": "12.1.6",
+    "nextra": "2.0.0-alpha.56",
+    "nextra-theme-docs": "2.0.0-alpha.59",
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
   },
   "devDependencies": {
-    "prettier": "^2.0.5"
```
Comment (B2o5T) [inline package.json:21]: why did you remove prettier from devDeps?
